crates/isograph_schema/src/add_pointers_to_supertypes.rs

```diff
@@ -1,10 +1,15 @@
+use std::vec;
```

Comment:
nit: this is already part of the prelude, i.e. doesn't need to be explicitly `use`d